src/lib/main.js

```diff
@@ -37,6 +39,8 @@ const defaultArchiveActions = [
   'upload-collect-forms',
   'upload-contact-forms',
   'upload-resources',
+  'upload-branding',
```

Comment:
@garethbowen Does it make sense to add the new commands here too? 🤔

Reply:
Yes! Good call.